specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/AddressCreate.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
How do you generate the example? From your real local test?

Reply:
Yes using Azure/oav